app/controllers/items_controller.rb

```diff
@@ -65,17 +74,28 @@ def destroy                                 #function that handles destruction o
   private
   
   def search                                  #function that find items matching the search phase.
-    if  params[:search].blank?
-      @items = nil
+    
+      
+    if  session[:search].blank?
+      @items = Item.all
     else
+      
+      
       # get item by that is related to search phrase
-      search_phrase = params[:search].downcase
-      item_by_author = Item.all.where("lower(author) LIKE :search", search: search_phrase)  
-      item_by_title = Item.all.where("lower(title) LIKE :search", search: search_phrase)  
-      item_by_description = Item.all.where("lower(description) LIKE :search", search: search_phrase)  
+      search_phrase = session[:search].downcase
+      #item_by_author = 
```

Comment:
Let's clean up these comments before merging! 🙏 It's not clear if these are intended to be fixed or removed.